.github/workflows/integration-tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        openshift: [v3.11.0]
+        openshift: [v3.9.0]
```

Comment:
I thought we were planning to get rid of `v3.9.0` . Why are we doing this?

Reply:
These action workflow is not the one in JKube repo. This was changes to reproduce and remove the thorntail error.

We need to agree on a valid environment matrix k8s and oc for the other repo and modify that workflow accordingly.

For this one I'd just change these versions whenever is necessary.